Godeps/LICENSES

```diff
@@ -35282,6 +35282,32 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 ================================================================================
+= vendor/github.com/glycerine/go-unsnap-stream licensed under: =
+
+Copyright (c) 2014 the go-unsnap-stream authors.
```

Comment:
This is not a standard license.  I'll reach out to the authors, but I am not sure we can use it.

Reply:
@thockin Were you able to determine anything? Can this license be used? or do I need to look for an alternative compression method.